index.js

```diff
@@ -1,2 +1,44 @@
    
- 
\ No newline at end of file
+const nav = document.querySelector("nav");
+const main = document.createElement("div");
+main.className = "mob-menu";
+nav.appendChild(main);
+
+main.innerHTML = `
+<img class="menu-list" src="images/Enabled.png" alt="menu">
+`;
+
+const menu = document.querySelector(".menu-list");
+menu.addEventListener("click" , () => {
+    const popup = document.createElement("div");
+    popup.className = "popup";
+    popup.innerHTML = `
+    <i class="fa-solid fa-xmark close"></i>
+    <ul class="menu-tags">
+    <li><a class="menu-link1 menu-portfolio" href="">Portfolio</a></li>
+    <li><a class="menu-link2 menu-about" href="#about">About</a></li>
+    <li><a class="menu-link3 menu-contact" href="#footer">Contact</a></li>
+      </ul>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_189](https://user-images.githubusercontent.com/69638013/168633407-b7cac435-6531-4b00-b57c-8d68e7acadc3.png)  |   ![Screenshot_190](https://user-images.githubusercontent.com/69638013/168633642-29f1daf9-3f14-4130-ba9c-0020cd91b606.png)

- [ ] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 
### OPTIONAL
- [ ] It would be better for the background not to be scrollable when the mobile nav is open 👍🏿